storm-client/src/jvm/org/apache/storm/executor/Executor.java

```diff
@@ -387,11 +387,13 @@ protected void setupTicks(boolean isSpout) {
                 LOG.info("Timeouts disabled for executor {}:{}", componentId, executorId);
             } else {
                 StormTimer timerTask = workerData.getUserTimer();
-                TupleImpl tuple = new TupleImpl(workerTopologyContext, new Values(tickTimeSecs),
-                    (int) Constants.SYSTEM_TASK_ID, Constants.SYSTEM_TICK_STREAM_ID);
-                final List<AddressedTuple> tickTuple =
-                    Lists.newArrayList(new AddressedTuple(AddressedTuple.BROADCAST_DEST, tuple));
-                timerTask.scheduleRecurring(tickTimeSecs, tickTimeSecs, () -> receiveQueue.publish(tickTuple));
+                timerTask.scheduleRecurring(tickTimeSecs, tickTimeSecs, () -> {
```

Comment:
Please put a comment on why it was changed with JIRA number as mentioned in 1.x PR [here](https://github.com/apache/storm/pull/2532#issuecomment-360382510)

Reply:
Addressed. Commit amended since it is a tiny change.